README.md

```diff
@@ -66,13 +66,13 @@ Time | Event
 09:00 | Arrive and Set Up
 09:15 | Start Hacking!
 10:30 | Coffee Break
-10:45 | More Hacking
+11:00 | More Hacking
 12:00 | Lunch Break
-13:15 | More Hacking
-14:30 | Coffee Break
-14:45 | More Hacking
-15:30 | Close out
-16:00 | End of Day 3
+13:30 | More Hacking
+15:30 | Coffee Break
+16:00 | More Hacking
+17:00 | Close out
+17:15 | End of Day 3
```

Comment:
do coffee break at 3-3:15pm and end day at 4:30pm? I think people are going to leave early, I myself have train at 5pm

Reply:
I didn't think about people leaving earlier, you're right.